packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -459,6 +461,19 @@ export class AccountResolver {
     return Account.findByUid(uid, options);
   }
 
+  @Query((returns) => EmailBounceStatusPayload, {
+    description: 'Check if bounces exist for an account, using email address.',
+  })
+  @CatchGatewayError
+  public async emailBounceStatus(
+    @Args('input', { type: () => String! })
+    input: string
+  ): Promise<EmailBounceStatusPayload> {
+    const bounces = await EmailBounce.findByEmail(input);
+    let hasHardBounce = bounces.some((bounce) => bounce.bounceType === 1);
```

Comment:
Nit: this can be a `const`. Also does the `return` need to return a copy with `{ ...{ hasHardBounce } };` or can we `return { hasHardBounce }`?